crates/cli/src/lib.rs

```diff
@@ -0,0 +1,62 @@
+//! The interpretation of the commands that the CLI cah handle.
+//!
+//! The CLI can do a few things. This provides a central point where those things are routed and
+//! then done, making it easier to test this crate deterministically.
```

Comment:
```suggestion
//! The interpretation of the commands that the CLI can handle.
//!
//! The CLI can do a few things. This provides a central point where those things are routed and
//! then done, making it easier to test this crate deterministically.
```

What is the purpose of the CLI? Is it only to initialize the config?

Reply:
Yes, for now. I added a bit more commentary to `main.rs`.